.travis.yml

```diff
@@ -5,3 +5,4 @@ install:
   - pip install -r requirements.txt
 sudo: enabled
 os: linux
+script: python welcome.py &
```

Comment:
This is going to make our server run on our Travis VM somewhere.  We should remove it and replace it with scripts to deploy to bluemix. Also, we should only deploy on master branch changes

Reply:
I will be working on the automatic deployment to bluemix. With this marge, this will allow automatically build. I will working on unit testing and automatically deployment.